### DISCUSSION TITLE
Sync files and subfolders

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1479] [Ideas]
I was happy to see that when uploading files in the asset manager, file versions were created in .versions

What would be extremely useful and time saving is if the Sync Files / Subfolders action created versions in .versions too

**Comment by meetmandeep** [comment]
This will required a dedicated thread since there is no cap on number of files and destination (Cloud). It's very likely such a process could take hours to complete. We will not be adding this to our roadmap at this time given the complexity to maintain such code.

**Reply by eswanzey** [reply]
Yes, totally understandable. It seems like a good opportunity for someone to create  a web service that would batch process files, same as what tinypng/tinyjpg does.
